### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by tanchak123**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -1,20 +1,25 @@
 package core.basesyntax;
 
 public class FootballMatch {
+     public static void main(String[] args) {
+        System.out.println(matchResult(1,2,1,1));
+    }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: don't create main in solutions. you can use it for tests, but don't forget to remove before commit

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
-     * если угадал, что выиграла какая-либо команда
-     * (пр. сыграли 1:2, а ставил 0:3 - то пользователь у выиграше)
-     * то возвращать 1
-     * если не угадал ничего - возвращать 0.</p>
-     */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public static int matchResult(int result1, int result2, int bet1, int bet2) {
+        int winr;
+        int betr;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: try to solve without creating variables. just "return statement;"

Comment (tanchak123) [reply src/main/java/core/basesyntax/FootballMatch.java]: +

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public static int matchResult(int result1, int result2, int bet1, int bet2) {
+        return (result1 == bet1 && result2 == bet2)
+                ? 2
+                : result1 < result2 && bet1 < bet2
+                || result1 > result2 && bet1 > bet2
+                || result1 == result2 && bet1 == bet2
+                ? 1
+                : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1 && result2 == bet2) ? 2 : 
        (result1 < result2 && bet1 < bet2) || 
        (result1 > result2 && bet1 > bet2) || 
        (result1 == result2 && bet1 == bet2) ? 1 : 0;
```
much prettier, isn't it?

Comment (Tarabarnik) [reply src/main/java/core/basesyntax/FootballMatch.java]: The code should be written in one style and easy to read

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**